components/Input/styles.css

```diff
@@ -1 +1,44 @@
 @import url(../../styles.css);
+
+.input {
+	border: 1px solid var(--theme-primary);
+	border-radius: 5px;
+	padding: 8px;
+	width: 100%;
+	outline: none;
+}
+
+.input:focus {
+	outline: 1px solid var(--theme-primary);
+}
+
+.input-text-area {
+	height: 5rem;
+}
+
+.form-container {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.form {
+	background-color: var(--theme-bg-white);
+	box-shadow: 2px 2px 5px 0px var(--theme-card-box-shadow);
+	border-radius: 8px;
+	max-width: 25rem;
+	padding: 0 2rem;
+}
+
+.form .input {
+	border: 1px solid var(--theme-error);
+	outline: 1px solid var(--theme-error);
+}
```

Comment:
If you are giving same values to both border and outline properties, Why not give just a border of 2px. Other than that you are good to merge.

Reply:
well this thickness was ideal. When I used 2px on border it was kind of thin and 2px on outline was kind of too thick. So I went with this